gemini/use-cases/retrieval-augmented-generation/multimodal_rag_QnA_usecase.ipynb

```diff
@@ -0,0 +1,882 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    url_prefix = "https://raw.githubusercontent.com/GoogleCloudPlatform/generative-ai/main/language/use-cases/document-qa/utils"
</pre><p>I'm not sure how many of these files are actually needed? I would recommend seeing if there's a way to refactor and use the vertex ai sdk directly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Refractor code to use vertex ai sdk for vector search related tasks.